content/enterprise_influxdb/v1.9/administration/manage/security/get-started-auth.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Get started with authentication and authorization
+description: >
+  Learn the basics of managing authentication and authorization in InfluxDB Enterprise
+menu:
+  enterprise_influxdb_1_9:
+    name: Get started with authentication and authorization
+    parent: Manage security
+weight: 30
+related:
+  - /enterprise_influxdb/v1.9/guides/fine-grained-authorization/
+  - /{{< latest "chronograf" >}}/administration/managing-influxdb-users/
+---
+
+To secure and manage access to an InfluxDB Enterprise cluster, consider the following two aspects:
+- *authentication* (verifying a user's identity)
+- *authorization* (verifying what the user has access to)
+
+{{% enterprise-warning-authn-b4-authz %}}
+
+## Permissions in InfluxDB Enterprise
+
+InfluxDB Enterprise has an [expanded set of 16 permissions](/enterprise_influxdb/v1.9/administration/manage/security/permissions/#permissions).
+These permissions allow for
+controlling read and write access to data for all databases at once,
+as well as permitting certain cluster-management actions like creating or deleting resources.
+
+InfluxB 1.x OSS only supports database-level privileges: `READ` and `WRITE`.
+A third permission, `ALL`, grants admin privileges.
+These can only be _can only be granted by using InfluxQL_.
+
+## Manage user authorization
+
+There are three ways to manage authorizations in InfluxDB Enterprise:
+
+- via [InfluxQL](#manage-read-and-write-privileges-with-influxql)
+- via [Chronograf](#manage-enterprise-permissions-with-chronograf)
+- via the [InfluxDB Enterprise meta API](#manage-enterprise-permissions-with-the-meta-api) (**Recommended**)
+
+### Manage read and write privileges with InfluxQL
+
+{{% note %}}
+InfluxQL can can only grant `READ`, `WRITE`, and `ALL PRIVILEGES` privileges.
+
+To use the full set of permissions, [use Chronograf](#manage-specific-privileges-with-chronograf) or the [Meta API (recommended)](#influxdb-enterprise-meta-api).
```

Comment:
let's move this up to "use one of the following" list to help folks make a choice